configs/train_configs/sft/tulu3_8b_preview_mix.yaml

```
@@ -0,0 +1,29 @@
+# Note, the exact model was trained on TPUs in a different repo
+# Example file for mixing, use exact tulu 2 mixture for replication allenai/tulu-v2-sft-mixture
+model_name_or_path: meta-llama/Meta-Llama-3-8B
+model_revision: main
+use_flash_attn: true
+tokenizer_name: meta-llama/Meta-Llama-3-8B
+use_slow_tokenizer: true
+dataset_mixer:
+  allenai/tulu-v2-sft-mixture-olmo-4096: 1.0
```

For none olmo configs :) 
```suggestion
  allenai/tulu-v2-sft-mixture: 1.0
```